docs/source/machine-learning.rst

```diff
@@ -0,0 +1,111 @@
+Machine Learning
+================
+
+Dask facilitates machine learning, statistics, and optimization workloads in a
+variety of ways.  Generally Dask tries to support other high-quality solutions
+within the PyData ecosystem rather than reinvent new systems.  Dask makes it
+easier to scale single-machine libraries like Scikit-Learn where possible and
+makes using distributed libraries like XGBoost or Tensorflow more comfortable
+for average users.
+
+In all cases you will have to install extra dependencies like scikit-learn_ to
+gain full advantage.  These dependencies are listed at the beginning of each
+section and are both conda installable from conda-forge or pip installable.
+
+Dask can be helpful in the following ways:
+
+SKLearn: Model selection
+------------------------
+
+*Extra Dependencies*: This depends on scikit-learn_ and dask-searchcv_.
+
+Dask can help to accelerate model selection and hyper parameter searches
+(GridSearchCV, RandomSearchCV) with dask-searchcv_.
+
+.. code-block:: python
+
+   from sklearn.feature_extraction.text import CountVectorizer
+   from sklearn.feature_extraction.text import TfidfTransformer
+   from sklearn.linear_model import SGDClassifier
+   from sklearn.pipeline import Pipeline
+
+   pipeline = Pipeline([('vect', CountVectorizer()),
+                        ('tfidf', TfidfTransformer()),
+                        ('clf', SGDClassifier())])
+
+   # from sklearn.model_selection import GridSearchCV  # use SKLearn
+   from dask_searchcv import GridSearchCV              # or use Dask
+
+   grid_search = GridSearchCV(pipeline, parameters)
+   grid_search.fit(data.data, data.target)
+
+
+SKLearn: Accelerated Joblib
+---------------------------
+
+*Extra Dependencies*: This depends on scikit-learn_ and distributed_.
+
+Scikit-Learn parallelizes its algorithms internally with Joblib_ a simple
+parallel computing API used throughout sklearn's codebase.  While Joblib
+normally runs from a local thread or processing pool Dask can also step in and
+provide distributed computing support.  See
+http://distributed.readthedocs.io/en/latest/joblib.html for more
+information.
+
+.. code-block:: python
+
+   import distributed.joblib
+   import joblib
+
+   with joblib.parallel_backend('dask.distributed', scheduler_host='localhost:8786'):
+       # normal sklearn code
+
+Convex Optimization Dask.Array
```

Comment:
This title feels like it's missing a word. "Convex Optimization with Dask.Array"? Any argument against the more descriptive "Generalized Linear Models"?